src/www/messages/pt-PT.json

```diff
@@ -14,10 +14,11 @@
   "disconnected-server-state": "Desligado",
   "disconnecting-server-state": "A desligar…",
   "email-feedback-input": "Endereço de email (opcional)",
+  "error-details": "Detalhes",
   "error-feedback-submission": "Lamentamos, mas não nos foi possível enviar os seus comentários. Verifique se tem ligação à Internet e tente novamente.",
   "error-invalid-access-key": "Chave de acesso inválida. Tente novamente ou envie comentários para obter ajuda.",
   "error-server-already-added": "O servidor “{serverName}” já foi adicionado.",
-  "error-server-incompatible": "Lamentamos, mas esta chave de acesso não é compatível com esta versão do Outline.",
+  "error-server-incompatible": "Lamentamos, mas esta token de acesso não é compatível com esta versão do Outline.",
```

Comment:
This is a regression. Not only the old term was better "chave de acesso", but now it's inconsistent. `error-invalid-access-key` is still using "chave de acesso".

Reply:
Let's keep `error-details` and revert the other one here.